src/PointEval/PointEvalHandler.jl

```diff
@@ -261,7 +266,7 @@ function _get_point_values!(
     # extract variables
     local_coords = ph.local_coords
     # preallocate some stuff specific to this cellset
-    pv = PointScalarValuesInternal(first(local_coords), ip)
+    pv = PointScalarValuesInternal(local_coords[findfirst(x->!isnothing(x), local_coords)], ip)
```

Comment:
Maybe just  return early if there is no found points?
```suggestion
    idx = findfirst(!isnothing, local_coords)
    idx === nothing && return out_vals
    pv = PointScalarValuesInternal(local_coords[idx], ip)
```